src/kube-runtime/src/runtime.d/runtime_entry.sh

```diff
@@ -96,6 +96,10 @@ echo "[INFO] USER COMMAND START"
 USER_PID=$!
 
 wait ${USER_PID}
+
+# flush cache to disk
+sync
```

Comment:
have you tested both stdout & stderr, w/ or w/o output?

Reply:
stdout & stderr tested.
with/without output tested.